dart/lib/src/transport/rate_limit.dart

```diff
@@ -0,0 +1,9 @@
+import 'rate_limit_category.dart';
+
+/// `RateLimit` containing limited `RateLimitCategory` and duration in milliseconds.
+class RateLimit {
+  RateLimit(this.category, this.durationInMillis);
+
+  final RateLimitCategory category;
+  final int durationInMillis;
```

Comment:
I wonder if it makes sense to use `Duration` here. I would prefer that over a simple int.
```suggestion
  final Duration duration;
```

See https://api.dart.dev/stable/2.12.2/dart-core/Duration-class.html

Is this the duration during which no new events from `category` should be sent to Sentry?

Reply:
Yeah, same data structure as in Java SDK.